Source/IGListAdapter.m

```diff
@@ -716,7 +716,7 @@ - (__kindof UICollectionViewCell *)dequeueReusableCellOfClass:(Class)cellClass
     IGParameterAssert(sectionController != nil);
     IGParameterAssert(cellClass != nil);
     UICollectionView *collectionView = self.collectionView;
-    IGAssert(collectionView != nil, @"Reloading adapter without a collection view.");
+    IGAssert(collectionView != nil, @"Tried to reload the adapter from %@ without a collection view at index %zi.", sectionController, index);
```

Comment:
Can this say something like `Dequeueing cell of class %@ without a collection view at section %zi`?